content/d3/concepts/styling/terms/text/text.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.text()'
+Description: 'This method is used to set or get text content of selected elements'
+Subjects:
+  - 'Data Visualization'
+  - 'Data Science'
+Tags:
+  - 'Methods'
+  - 'Data'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/computer-science'
+---
+
+The **`.text()`** method in D3 is used to set text content for any selected element in a document, this method can also be used to get the value of any selected element. This method can take a single value as a argument or a calback function for a complex text content manipulation of a selected element
+
+## Syntax
+
+```pseudo
+    d3.selection("element")                 // A selection method is called on an element
+    .text("Your text content goes here");   // A text method is called and the text content 
+                                            // is passed to it as an argument
```

Comment:
instead of adding these comments in the pseudo block create bullets point down below instead.